### ISSUE TITLE
Error creating cluster when # of NAT GWs exceeds limits

### ISSUE THREAD

**Issue by mreferre** [weaveworks/eksctl#1646]
I have been reported an issue from a customer that encounter this error when trying to deploy a new EKS cluster with Fargate enabled (this is possibly unrelated):

```
The maximum number of addresses has been reached. (Service: AmazonEC2; Status Code: 400; Error Code: AddressLimitExceeded
```
This happens with EKSCTL 0.11.

I have not yet had the time to reproduce it yet but it looks like the problem is due to the max number of NAT GWs in the account being reached (default limits are documented [here](https://docs.aws.amazon.com/vpc/latest/userguide/amazon-vpc-limits.html)).

This needs further investigation but I am opening it now to track it on our radars.

**Comment by martina-if** [comment]
Hi @mreferre this seems like a quota limit being hit, do you think there's anything that `eksctl`'s could do to avoid this?

**Comment by mreferre** [comment]
Hi @martina-if . I had to go back and re-assemble the internal thread around this (old issue). I believe that the conclusion I reached when I recreated it is that the text below was only partial and the whole test was: 
```
[✖]  AWS::EC2::EIP/NATIP: CREATE_FAILED – "The maximum number of addresses has been reached. (Service: AmazonEC2; Status Code: 400; Error Code: AddressLimitExceeded; Request ID: 6826fecd-0d28-4efe-b28f-041d8359e922)"
```

I don't know if the error reporting got better in the meanwhile or if who reported it in the first place only copied/pasted the last part but it seems to be a non-problem at this point (I don't know why I did not come here to close the issue - sorry about that. I am closing it now.
